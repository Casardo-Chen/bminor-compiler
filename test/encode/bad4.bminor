"hello \0xFG world"