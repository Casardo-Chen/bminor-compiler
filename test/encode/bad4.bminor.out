Input String    : "hello \0xFG world"
Error: Invalid hexadecimal escape sequence
Invalid String: "hello \0xFG world"
